src/NUnitEngine/nunit.engine.core/RuntimeFramework.cs

```diff
@@ -204,6 +204,8 @@ private Version GetClrVersionForFramework(Version frameworkVersion)
                             return new Version(6, 0, 0);
                         case 7:
                             return new Version(7, 0, 0);
+                        case 8:
+                            return new Version(8, 0, 0);
```

Comment:
Instead of doing this every year, can we not have a default case where version is passed through.

Reply:
I'd agree with this. It seems silly to keep adding a new version and passing in the case value.

Reply:
Agree, I'll fix